recipes/exiv2/all/patches/0001-link.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index c52b352f..2c270c5f 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -215,7 +215,7 @@ else()
+ endif()
+ 
+ if( EXIV2_ENABLE_PNG )
+-    target_link_libraries( exiv2lib PRIVATE $<BUILD_INTERFACE:${ZLIB_LIBRARIES}>)
```

Comment:
Should we make this patch also wipe the lines like
```
target_include_directories(exiv2lib_int PRIVATE ${ZLIB_INCLUDE_DIR})
```
So we ensure that we're using the headers exposed by `ZLIB::ZLIB`?

Reply:
I think `FindZLIB.cmake`, generated by `cmake_find_package` defines `ZLIB_INCLUDE_DIR`.
This means the zlib include directories are added twice.
This is no problem as cmake filters duplicate paths away.